src/ui/MessageStatus/index.tsx

```diff
@@ -21,7 +18,7 @@ export const MessageStatusTypes = OutgoingMessageStates;
 
 interface MessageStatusProps {
   className?: string;
-  message: UserMessage | FileMessage;
+  message: SendableMessageType;
```

Comment:
channel.lastMessage 도 받아서 Null 과 CoreMessageType 으로 바뀌어야 할 것 같습니당

```suggestion
  message: CoreMessageType | null;
```

Reply:
nice catch!